otelfiber/fiber.go

```diff
@@ -49,11 +81,16 @@ func Middleware(service string, opts ...Option) fiber.Handler {
 		c.Locals(tracerKey, tracer)
 		savedCtx, cancel := context.WithCancel(c.UserContext())
 
+		start := time.Now()
+		httpServerActiveRequests.Add(savedCtx, 1)
 		defer func() {
+			httpServerDuration.Record(savedCtx, float64(time.Since(start).Microseconds())/1000)
+			httpServerRequestSize.Record(savedCtx, int64(len(c.Request().Body())))
+			httpServerResponseSize.Record(savedCtx, int64(len(c.Response().Body())))
+			httpServerActiveRequests.Add(savedCtx, -1)
```

Comment:
shouldn't [some attributes](https://opentelemetry.io/docs/reference/specification/metrics/semantic_conventions/http-metrics/#attributes) be here?